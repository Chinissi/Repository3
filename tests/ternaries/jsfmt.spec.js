run_spec(__dirname, ["flow", "typescript", "babel"]);
run_spec(__dirname, ["flow", "typescript", "babel"], { tabWidth: 4 });
run_spec(__dirname, ["flow", "typescript", "babel"], { useTabs: true });
run_spec(__dirname, ["flow", "typescript", "babel"], {
  useTabs: true,
  tabWidth: 4
});
