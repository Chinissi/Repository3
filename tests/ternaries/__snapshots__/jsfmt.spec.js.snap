// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nested.js 1`] = `
"const value = condition1
  ? value1
  : condition2
      ? value2
      : condition3
          ? value3
          : value4;

const value =
  condition1 ? value1 :
  condition2 ? value2 :
  condition3 ? value3 :
  value4;
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
const value = condition1
  ? value1
  : condition2 ? value2 : condition3 ? value3 : value4;

const value = condition1
  ? value1
  : condition2 ? value2 : condition3 ? value3 : value4;
"
`;

exports[`nested.js 2`] = `
"const value = condition1
  ? value1
  : condition2
      ? value2
      : condition3
          ? value3
          : value4;

const value =
  condition1 ? value1 :
  condition2 ? value2 :
  condition3 ? value3 :
  value4;
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
const value = condition1
    ? value1
    : condition2 ? value2 : condition3 ? value3 : value4;

const value = condition1
    ? value1
    : condition2 ? value2 : condition3 ? value3 : value4;
"
`;

exports[`nested.js 3`] = `
"const value = condition1
  ? value1
  : condition2
      ? value2
      : condition3
          ? value3
          : value4;

const value =
  condition1 ? value1 :
  condition2 ? value2 :
  condition3 ? value3 :
  value4;
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
const value = condition1
  ? value1
  : condition2 ? value2 : condition3 ? value3 : value4;

const value = condition1
  ? value1
  : condition2 ? value2 : condition3 ? value3 : value4;
"
`;

exports[`test.js 1`] = `
"const obj0 = conditionIsTruthy ? shortThing : otherShortThing

const obj1 = conditionIsTruthy ? { some: 'long', object: 'with', lots: 'of', stuff } : shortThing

const obj2 = conditionIsTruthy ? shortThing : { some: 'long', object: 'with', lots: 'of', stuff }

const obj3 = conditionIsTruthy ? { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff } : shortThing

const obj4 = conditionIsTruthy ? shortThing : { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff }

const obj5 = conditionIsTruthy ? { some: 'long', object: 'with', lots: 'of', stuff } : { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff }
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
const obj0 = conditionIsTruthy ? shortThing : otherShortThing;

const obj1 = conditionIsTruthy
  ? { some: \\"long\\", object: \\"with\\", lots: \\"of\\", stuff }
  : shortThing;

const obj2 = conditionIsTruthy
  ? shortThing
  : { some: \\"long\\", object: \\"with\\", lots: \\"of\\", stuff };

const obj3 = conditionIsTruthy
  ? {
      some: \\"eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger\\",
      object: \\"with\\",
      lots: \\"of\\",
      stuff
    }
  : shortThing;

const obj4 = conditionIsTruthy
  ? shortThing
  : {
      some: \\"eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger\\",
      object: \\"with\\",
      lots: \\"of\\",
      stuff
    };

const obj5 = conditionIsTruthy
  ? { some: \\"long\\", object: \\"with\\", lots: \\"of\\", stuff }
  : {
      some: \\"eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger\\",
      object: \\"with\\",
      lots: \\"of\\",
      stuff
    };
"
`;

exports[`test.js 2`] = `
"const obj0 = conditionIsTruthy ? shortThing : otherShortThing

const obj1 = conditionIsTruthy ? { some: 'long', object: 'with', lots: 'of', stuff } : shortThing

const obj2 = conditionIsTruthy ? shortThing : { some: 'long', object: 'with', lots: 'of', stuff }

const obj3 = conditionIsTruthy ? { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff } : shortThing

const obj4 = conditionIsTruthy ? shortThing : { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff }

const obj5 = conditionIsTruthy ? { some: 'long', object: 'with', lots: 'of', stuff } : { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff }
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
const obj0 = conditionIsTruthy ? shortThing : otherShortThing;

const obj1 = conditionIsTruthy
    ? { some: \\"long\\", object: \\"with\\", lots: \\"of\\", stuff }
    : shortThing;

const obj2 = conditionIsTruthy
    ? shortThing
    : { some: \\"long\\", object: \\"with\\", lots: \\"of\\", stuff };

const obj3 = conditionIsTruthy
    ? {
          some: \\"eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger\\",
          object: \\"with\\",
          lots: \\"of\\",
          stuff
      }
    : shortThing;

const obj4 = conditionIsTruthy
    ? shortThing
    : {
          some: \\"eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger\\",
          object: \\"with\\",
          lots: \\"of\\",
          stuff
      };

const obj5 = conditionIsTruthy
    ? { some: \\"long\\", object: \\"with\\", lots: \\"of\\", stuff }
    : {
          some: \\"eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger\\",
          object: \\"with\\",
          lots: \\"of\\",
          stuff
      };
"
`;

exports[`test.js 3`] = `
"const obj0 = conditionIsTruthy ? shortThing : otherShortThing

const obj1 = conditionIsTruthy ? { some: 'long', object: 'with', lots: 'of', stuff } : shortThing

const obj2 = conditionIsTruthy ? shortThing : { some: 'long', object: 'with', lots: 'of', stuff }

const obj3 = conditionIsTruthy ? { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff } : shortThing

const obj4 = conditionIsTruthy ? shortThing : { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff }

const obj5 = conditionIsTruthy ? { some: 'long', object: 'with', lots: 'of', stuff } : { some: 'eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger', object: 'with', lots: 'of', stuff }
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
const obj0 = conditionIsTruthy ? shortThing : otherShortThing;

const obj1 = conditionIsTruthy
  ? { some: \\"long\\", object: \\"with\\", lots: \\"of\\", stuff }
  : shortThing;

const obj2 = conditionIsTruthy
  ? shortThing
  : { some: \\"long\\", object: \\"with\\", lots: \\"of\\", stuff };

const obj3 = conditionIsTruthy
  ? {
      some: \\"eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger\\",
      object: \\"with\\",
      lots: \\"of\\",
      stuff
    }
  : shortThing;

const obj4 = conditionIsTruthy
  ? shortThing
  : {
      some: \\"eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger\\",
      object: \\"with\\",
      lots: \\"of\\",
      stuff
    };

const obj5 = conditionIsTruthy
  ? { some: \\"long\\", object: \\"with\\", lots: \\"of\\", stuff }
  : {
      some: \\"eeeeeeeeeeeeven looooooooooooooooooooooooooooooonger\\",
      object: \\"with\\",
      lots: \\"of\\",
      stuff
    };
"
`;
